hail_search/test_utils.py

```diff
@@ -418,8 +423,534 @@
+                "familyGuid": "F000005_5",
+                "individualGuid": "I00001_bon_b15_95_1_d1",
+                "numAlt": 1,
+                "dp": 49,
+                "gq": 99,
+                "ab": 0.5306122303009033,
+                'filters': [],
+            },
+        ],
+        "I00003_bon_b15_95_3_d1": [
+            {
+                "sampleId": "BON_B15-95_3_D1",
+                "sampleType": "WES",
+                "familyGuid": "F000005_5",
+                "individualGuid": "I00003_bon_b15_95_3_d1",
+                "numAlt": 1,
+                "dp": 78,
+                "gq": 20,
+                "ab": 0.4743589758872986,
+                'filters': [],
+            }
+        ],
+        "I00004_bon_b15_95_4_d1": [
+            {
+                "sampleId": "BON_B15-95_4_D1",
+                "sampleType": "WES",
+                "familyGuid": "F000005_5",
+                "individualGuid": "I00004_bon_b15_95_4_d1",
+                "numAlt": 0,
+                "dp": None,
+                "gq": 40,
+                "ab": None,
+                'filters': [],
+            }
+        ],
+    },
+    "populations": {
+        "seqr": {"af": 0.0, "ac": 0, "an": 0, "hom": 0},
+        "topmed": {"af": 0.0, "ac": 0, "an": 0, "hom": 0, "het": 0},
+        "exac": {
+            "af": 0.0,
+            "ac": 0,
+            "an": 0,
+            "hom": 0,
+            "hemi": 0,
+            "het": 0,
+            "filter_af": 0.0,
+        },
+        "gnomad_exomes": {
+            "af": 0.0,
+            "ac": 0,
+            "an": 0,
+            "hom": 0,
+            "hemi": 0,
+            "filter_af": 0.0,
+        },
+        "gnomad_genomes": {
+            "af": 0.0,
+            "ac": 0,
+            "an": 0,
+            "hom": 0,
+            "hemi": 0,
+            "filter_af": 0.0,
+        },
+    },
+    "predictions": {
+        "cadd": None,
+        "eigen": None,
+        "mpc": None,
+        "primate_ai": None,
+        "splice_ai": None,
+        "splice_ai_consequence": None,
+        "mut_taster": None,
+        "polyphen": None,
+        "revel": None,
+        "sift": None,
+        "fathmm": None,
+        "mut_pred": None,
+        "vest": None,
+        "gnomad_noncoding": None,
+    },
+    "chrom": "2",
+    "pos": 44312654,
+    "ref": "T",
+    "alt": "C",
+    "mainTranscriptId": None,
+    "selectedMainTranscriptId": None,
+    "familyGuids": ["F000005_5"],
+    "variantId": None,
+    "liftedOverGenomeVersion": None,
+    "liftedOverChrom": None,
+    "liftedOverPos": None,
+    "clinvar": None,
+    "hgmd": None,
+    "screenRegionType": None,
+    "transcripts": {},
+    "sortedMotifFeatureConsequences": None,
+    "sortedRegulatoryFeatureConsequences": None,
+    "_sort": [None],
+    "genomeVersion": "38",
+}
+
+FAMILY_5_SAMPLE_DATA = {
+    'SNV_INDEL': [
+        {'sample_id': 'BON_B15-95_1_D1', 'individual_guid': 'I00001_bon_b15_95_1_d1', 'family_guid': 'F000005_5',
```

Comment:
As above, do we really need a new family for this or could we add additional sample type data for one of the existing families?